src/main/java/seedu/taskify/logic/parser/ParserUtil.java

```diff
@@ -34,9 +43,52 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         if (!StringUtil.isNonZeroUnsignedInteger(trimmedIndex)) {
             throw new ParseException(MESSAGE_INVALID_INDEX);
         }
+
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
+    /**
+     * Parses {@code oneBasedIndexes} into a list of {@code Index} and returns it.
+     * @param oneBasedIndexes user's input excluding the command word
+     * @return a list of {@code Index} representing all valid indexes in {@code oneBasedIndexes}
+     * @throws ParseException if {@code oneBasedIndexes} cannot be parsed properly
+     */
+    public static List<Index> parseMultipleIndex(String oneBasedIndexes) throws ParseException {
+        assert hasMultipleValidIndex(oneBasedIndexes) : "ParserUtil#parseMultipleIndex should not be called on this "
+                + "string";
+        String[] arguments = extractStringArgumentsIntoIndexes(oneBasedIndexes);
+        boolean hasOnlyOneArgument = arguments.length == 1;
+
+        if (hasOnlyOneArgument) {
+            // throw new AssertionError instead?
```

Comment:
I think exceptions are more suitable as we are trying to figure out errors with user input

Reply:
Sorry, can I clarify which line you are referring to?

Reply:
I was commenting on your comment on line 63 haha